challenge_3/elixir/joegotflow83/majority_element/lib/majority_element.ex

```diff
@@ -0,0 +1,26 @@
+defmodule MajorityElement do
+  def find_major_ele(input) when is_list(input) do
+    input
+    |> Enum.reduce(%{}, &count/2)
+    |> Map.to_list
+    |> determine_majority(length(input))
+  end
+
+  def determine_majority([head | tail], n) do
+    {num, value} = head
+    cond do
+      value > (n / 2) ->
+        num
+      true ->
```

Comment:
Good to know. I'm still wrapping my head around `case` vs `cond`. I would have expected `_` to be a truthy value, akin to a final `else` clause in an if stack or a `default` in a switch stack.